set.mm

```diff
@@ -231,13 +231,14 @@ surrounded by whitespace (space, tab, CR, LF, or FF).
       file or has been already been inserted, it will not be inserted again.
 
 Inside of comments, it is recommended that labels be preceded with a tilde (~)
-and math symbol tokens be enclosed in grave accents (` `).  This way the LaTeX
+and math symbol tokens be enclosed in backticks (` `).  These tildes, tokens,
```

Comment:
The official name of U+0060 in the Unicode charts is **grave accent** (https://unicode.org/charts/PDF/U0000.pdf).  So I wonder whether replacing it with backtick is a good idea.  At least I would introduce it with its official name, then continuing with sth like 'or backtick, as we think is more common, and used in this document'

Reply:
I followed https://en.wikipedia.org/wiki/Backtick, which also mentions "grave accent" and others but uses "backtick" as the main word. To me, "grave accent" is, like other accents, a diacritical mark https://en.wikipedia.org/wiki/Grave_accent : it is the same symbol, but it takes that name when combined with a letter, as in àèù (to mention only the ones on my keyboard), like the German umlaut.

Reply:
Wikipedia has this article https://en.wikipedia.org/wiki/Grave_accent, too.  The accent put on letters has its own code U+0300 and is officially called combining grave accent.

Reply:
The official name is "grave accent", but I think we should say "backtick", because @benjub is right, a lot of people will be confused if we use the official name.

Reply:
Is there really someone who got it wrong in the past? Ok then, why not : ... enclosed in backticks (U+0060).

Reply:
I still think that official and scientific documentation should adhere to (thus support) standards where possible.  Sth along the lines of my second comment here would meet my expectations.  But if a majority thinks you can do completely without it, I will no longer object.

Reply:
No strong opinion. Anyone ? @avekens ? @tirix ?

Reply:
I'd say whatever consensus is. However, I suggest including both terms, one as the "main term" and the other as an "alternate term" just to make sure people will know what's meant.

E.g.:

> and math symbol tokens be enclosed in grave accents aka backticks (` `).  This way the LaTeX ...

Reply:
For me, David's suggestion (also known as) is still preferable.  But since the character is directly shown afterwards, I will go with that solution.  A similar discussion like this one is found here https://stackoverflow.com/questions/7857278/what-is-the-meaning-of-grave-accent-aka-backtick-quoted-characters-in-mysql.

Reply:
I wrote both

Reply:
Including both names is probably the best.
I prefer "aka" to "or", because "or" might let someone believe there are two alternatives, actually it's only different names for one and the same character.